makefile

```diff
@@ -168,4 +168,4 @@ clean-deps:
 clean-all: clean clean-doxygen clean-deps clean-libraries
 	@echo '  removing generated test files'
 	$(shell find test/prob -name '*_generated_*_test.cpp' -type f -exec rm {} +)
-	$(RM) $(wildcard test/gtest.o test/libgtest* test/prob/generate_tests$(EXE))
+	$(RM) $(wildcard test/gtest.o test/libgtest* test/prob/generate_tests$(EXE))
```

Comment:
Not sure what's happening here. It looks as though the exact same line code was deleted and added. I'm guessing GitHub is comparing revisions, as oppose to final code and current dev version.